src/command.rs

```diff
@@ -121,6 +123,30 @@ impl<'l, const C: usize> TryFrom<&'l Data<C>> for Validate<'l> {
     }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub struct VerifyCode<'l> {
+    pub label: &'l [u8],
+    pub response: &'l [u8],
+}
+
+impl<'l, const C: usize> TryFrom<&'l Data<C>> for VerifyCode<'l> {
+    type Error = Status;
+    fn try_from(data: &'l Data<C>) -> Result<Self, Self::Error> {
+        use flexiber::TaggedSlice;
+        let mut decoder = flexiber::Decoder::new(data);
+
+        let first: TaggedSlice = decoder.decode().unwrap();
+        assert!(first.tag() == (oath::Tag::Name as u8).try_into().unwrap());
+        let label = first.as_bytes();
+
+        let slice: TaggedSlice = decoder.decode().unwrap();
+        assert!(slice.tag() == (oath::Tag::Response as u8).try_into().unwrap());
+        let response = slice.as_bytes();
+
+        Ok(VerifyCode { label, response })
+    }
```

Comment:
This function should return errors rather than unwrapping and asserting

Reply:
Indeed. To be moved to a separate ticket. This specific implementation is just following general convention in this project.

Reply:
I think it shouldn't be possible to trigger a panic with invalid input.